sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCScanBuilder.scala

```diff
@@ -72,6 +72,9 @@ case class JDBCScanBuilder(
 
   private var pushedGroupByCols: Option[Array[String]] = None
 
+  override def supportCompletePushDown: Boolean =
+    jdbcOptions.numPartitions.map(_ == 1).getOrElse(true)
```

Comment:
In the case of multiple partitions, if partition columns are the same as group by columns, should `supportCompletePushDown` be set to true as well?

Reply:
I will followup a PR. @huaxingao Thanks.